docs/_docs/persistence/snapshots.adoc

```diff
@@ -160,27 +166,27 @@ the topology and wait while the data gets rebalanced and indexes are rebuilt.
 == Consistency Guarantees
 
 All snapshots are fully consistent in terms of concurrent cluster-wide operations as well as ongoing changes with Ignite
-Persistence and other files on nodes.
+Persistence and files on nodes.
 
 The cluster-wide snapshot consistency is achieved by triggering the link:https://cwiki.apache.org/confluence/display/IGNITE/%28Partition+Map%29+Exchange+-+under+the+hood[Partition-Map-Exchange]
 procedure. By doing that, the cluster will eventually get to the point in time when all previously started transactions are completed, and new
-ones are paused. Once this happens, the cluster initiates the snapshot creation procedure.
+ones are paused. Once this happens, the cluster initiates the snapshot creation procedure. Primary and backup cache partitions
+will also be inclued and fully consistent in created snapshots since the PME procedure also awaits transaction completion
+on backup cache partitions too.
 
-The consistency between the primary Ignite Persistence files and their snapshot copies is achieved by copying the primary
+The consistency between the Ignite Persistence files and their snapshot copies is achieved by copying the original
 files to the destination snapshot directory with tracking all concurrent ongoing changes. The tracking of the changes
-might require extra space on your storage media.
-
+might require extra space on the Ignite Persistence storage media (up to the current size of it).
```

Comment:
"up to the current size of it" - what is the "it"? Are you talking about the size of the persistence files or the storage media as a whole? Please clarify this in place.

Reply:
Fixed.